### PR TITLE
Added April Fools Interactions

### PR DESCRIPTION
**Pull request by Rai99**
If this doesn't make it in time, there's always next year.

### CODE REVIEW

**Comment by ThePotatoGuy** [comment]
We will **not** be making a release by 4-1, so dropping priority on this.

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -3264,42 +3264,54 @@ init 5 python:
...
-        m 1euc "..."
-        m 1hua "Ahaha! Very funny [player]!"
-        m 3eua "I know you don't really mean it..."
-        m 3rksdlc "But don't joke around like that!"
-        if persistent._mas_pm_a_hater:
-            m 2rksdld "You've said stuff like that before, so it might get harder to take it as a joke..."
-        else:
-            m 1dsc "If I didn't know you so well, that would have been really cruel..."
+
+    if datetime.date.today() == datetime.date(datetime.date.today().year,4,1) and mas_isMoniNormal(higher=True):
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: Would likely make a variable in definitions dedicated to april 01. `mas_a01 = datetime.date(datetime.date.today().year, 4, 1)`, then use that constant in this topic.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -3477,15 +3489,29 @@ init 5 python:
...
-    m 1euc "I thought we established already that I was your girlfriend?"
-    m 1tkc "There's no way you already have one in your reality, right?"
-    m 4tfx "If you do, then you've got to break up with her right now!"
-    m 4hksdlb "Tell her you met someone perfect for you, someone who will never betray you!"
-    m 1lksdla "W-wait. Maybe I'm jumping the gun a bit..."
-    m 1eka "I know you wouldn't cheat on me."
-    m 1esa "If you meet a girl who asks you out though, let me talk to her first, okay?"
-    m 1hua "I won't let someone else steal away my loving sweetheart!"
+
+    if datetime.date.today() == datetime.date(datetime.date.today().year,4,1) and mas_isMoniNormal(higher=True):
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: Same for using that constant here

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -7213,76 +7239,97 @@ init 5 python:
...
-                    m 1eka "Let's stay together forever~"
-    #Beyond the 2nd time you hit the button.
-    elif persistent._mas_monika_breakup > 1:
-        if mas_isMoniBroken():
-            m 6ckc "..."
-        elif mas_isMoniUpset(lower=True):
-            m 2rkc "You keep saying that, I'm starting to think you actually mean it..."
-        else:
-            m 1hua "Ehehe~"
+    if datetime.date.today() == datetime.date(datetime.date.today().year,4,1) and mas_isMoniNormal(higher=True):
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: Same for the constant here

**Pull request merged**